v1/lib/v1/searchable/query.rb

```diff
@@ -202,8 +292,17 @@ def self.date_range_queries(params)
       end
 
       private
+
+      ##
+      # Remove substring ".not_analyzed" from end of each field name in the
+      # given array.
+      #
+      # This method would be better named `analyzed_fields` or inlined into
```

Comment:
IIRC this method is only used in the one instance of compound field, for the work on https://issues.dp.la/issues/7596